nsot/api/serializers.py

```diff
@@ -149,8 +145,17 @@ def to_internal_value(self, data):
             'NsotSerializer.to_internal_value() data [before] = %r', data
         )
 
-        if 'site_id' not in data and 'site_pk' in kwargs:
-            data['site_id'] = kwargs['site_pk']
+        # FIXME(jathan): This MUST be ripped out once we migrate to V2 API and
+        # move away from the "site_id" field on pre-1.0 objects.
+        site_fields = ['site_id', 'site']
+        for site_field in site_fields:
```

Comment:
So just to confirm my understanding, `site_field` will be set via this `for` loop for its use in line 157? 

suggestion: Setting `site_field` more explicitly may help make this section a little easier to understand.

Reply:
Yeah using the `for..else` Pythonism there where iterator values will always remain. So in the case where it matches the `break` happens and the current value of `site_field` is  used, otherwise the iterator is exhausted and the `else` is fired and the explicit `site_field` value is used.